### PR TITLE
pv right sizing docs

### PR DESCRIPTION
**Pull request by nickcurie**
Basic docs for `savings/persistentVolumeSizing` API endpoint

### CODE REVIEW

**Review by bstuder99 — APPROVED** [review]
LGTM

File: api-persistent-volume-right-sizing.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        "clusterId": "cluster-one",
+        "maxUsageBytes": 206999552,
+        "averageUsageBytes": 202643394.05432096,
+        "recommendedCapacityBytes": 227699507.20000002,
+        "recommendedCostMonthly": 0.008626238289525955,
+        "currentCapacityBytes": 33787076061.866665,
+        "currentCostMonthly": 1.28,
+        "savingsMonthly": 1.1937376171047405,
+        "storageClass": "standard"
+```
```
Comment (srpomeroy) [inline api-persistent-volume-right-sizing.md:56]: Cna you please add the following metadata to the last line of the doc.
`<!--- {"article":"","section":"8454342470679","permissiongroup":"1500001277122"} --->`

**Review by srpomeroy — CHANGES_REQUESTED** [review]
Looks good @nickcurie.

One tweak for our docs platform.
Additionally, is this doc tied to v1.98 or can it be released now?

**Comment by nickcurie** [comment]
This is a v1.98 feature, so this doc should go out with that.

**Review by srpomeroy — APPROVED** [review]

**Comment by bstuder99** [comment]
Just checking in, we good to roll this out now?

**Comment by nickcurie** [comment]
Yep, this is live in 1.98 now!

**Pull request merged**